### PR TITLE
Rwc2017

### PR DESCRIPTION
**Pull request by MatthijsBurgh**

### CODE REVIEW

File: parameters/hardware/actuators/spindle_controller.ops [hunk 1]
```diff
@@ -82,12 +82,12 @@ SPINDLE_Safety.Nm						= N
 SPINDLE_Safety.maxJointErrors			= MAXERR
 SPINDLE_Safety.motorSaturations			= array(100.0)
 SPINDLE_Safety.maxConSatTime			= 1.0
-SPINDLE_Safety.additional_safeties		= strings("endswitch")
+#SPINDLE_Safety.additional_safeties		= strings("endswitch")
```
Comment (MatthijsBurgh) [inline parameters/hardware/actuators/spindle_controller.ops:85]: @MaxBaeten What is the status of the endswitch?

Comment (MaxBaeten) [reply parameters/hardware/actuators/spindle_controller.ops:85]: Should be fixed. Replace cabling or sensor. It is on Henk's hardware list. We should definitely not continue with this hack in place.

Comment (MatthijsBurgh) [reply parameters/hardware/actuators/spindle_controller.ops:85]: So merge this PR and create issue? Or revert this hack, so we are reminded that the sensor is not working?

**Review by Rayman — APPROVED** [review]
For the hardware changes, separate issues have been made

**Pull request merged**